backend/app/controllers/comments_controller.rb

```diff
@@ -12,6 +12,13 @@ def create
     end
   end
 
+  def destroy
+    memo = Memo.find(params[:memo_id])
+    comment = memo.comments.find(params[:id])
```

Comment:
Performant/Simple

以下のように書くとクエリの発行数が2回から1回になって少しだけパフォーマンスがいいと思います。
```suggestion
    comment = \
      Comment.find_by(id: params[:id], memo_id: params[:memo_id])
```